consensus/enclave/api/src/lib.rs

```diff
@@ -100,6 +101,83 @@ impl From<&Tx> for WellFormedTxContext {
     }
 }
 
+/// Defines a sort order for transactions in a block.
+/// Transactions are sorted by fee (high to low), then by transaction hash and any other fields.
+impl Ord for WellFormedTxContext {
+    fn cmp(&self, other: &Self) -> Ordering {
+        if self.fee != other.fee {
+            // Sort by fee, descending.
+            other.fee.cmp(&self.fee)
```

Comment:
Isn't this already the behavior when `fee` is the first field in the struct?

Reply:
Not quite. Deriving `Ord` with `fee` as the first field means that WellFormedTxContext is sorted by fee in ascending order, which is the opposite of what we want.